CHANGELOG.md

```diff
@@ -25,6 +31,10 @@ and this project adheres to
   the `start_time_us` and `start_time_cpu_us` metrics. These parameters were
   never meant to be used by end customers, and we recommend doing any such time
   adjustments outside Firecracker.
+- Booting with microVM kernels that rely on MPTable on x86_64 is deprecated and
+  support will be removed in v2.0 or later. We suggest to users of Firecracker
+  to use guest kernels with ACPI support. ACPI will be the only way Firecracker
```

Comment:
nit: ACPI will be the only way `on x86_64`
Also, can we add a link to the doc/section which points to the configs they'll need to enable for ACPI?

Reply:
Fixed both.